spaces/domains.json

```
@@ -1,5 +1,6 @@
 {
   "dao.anuinitiative.org": "daotest.anuinitiative.eth",
+  "vote.daocred.xyz": "daocred.eth"
```

A comma is missing at the end of the line of your sub-domain.
```suggestion
  "vote.daocred.xyz": "daocred.eth",
```